packages/webapp/src/containers/Insights/index.js

```diff
@@ -111,13 +111,14 @@ class Insights extends Component {
 
   generateView(cropNutritionalData, soilOMData, labourHappinessData, biodiversityData, pricesData, waterBalanceData, nitrogenBalanceData) {
     const insightData = {};
+    const isImperial = this.props.farm?.units?.measurement === 'imperial';
     insightData['PeopleFed'] = cropNutritionalData.preview + " meals";
     insightData['SoilOM'] = (soilOMData.preview || '0') + "%";
     insightData['LabourHappiness'] = labourHappinessData.preview ? labourHappinessData.preview + "/5" : 'Unavailable';
     insightData['Biodiversity'] = biodiversityData.preview + " species";
     insightData['Prices'] = pricesData.preview ? pricesData.preview + "% of market" : "Unavailable";
-    insightData['WaterBalance'] = waterBalanceData.preview + " mm";
-    insightData['NitrogenBalance'] = nitrogenBalanceData.preview + " kg";
+    insightData['WaterBalance'] = isImperial? Number(waterBalanceData.preview)*0.0393701 + " in": waterBalanceData.preview + " mm";
+    insightData['NitrogenBalance'] = isImperial? Number(nitrogenBalanceData.preview)*2.20462 + " lbs" : nitrogenBalanceData.preview + " kg";
```

Comment:
Same with this 2 constants, let's move them to the start of the file so it's clear what they represent